src/main/java/com/mongodb/jdbc/MongoPreparedStatement.java

```diff
@@ -4,32 +4,18 @@
 import java.io.Reader;
 import java.math.BigDecimal;
 import java.net.URL;
-import java.sql.Array;
-import java.sql.Blob;
-import java.sql.Clob;
-import java.sql.Date;
-import java.sql.NClob;
-import java.sql.ParameterMetaData;
-import java.sql.PreparedStatement;
-import java.sql.Ref;
-import java.sql.ResultSet;
-import java.sql.ResultSetMetaData;
-import java.sql.RowId;
-import java.sql.SQLException;
-import java.sql.SQLFeatureNotSupportedException;
-import java.sql.SQLXML;
-import java.sql.Time;
-import java.sql.Timestamp;
+import java.sql.*;
 import java.util.Calendar;
 
-public class MongoPreparedStatement extends MongoStatement implements PreparedStatement {
+public class MongoPreparedStatement implements PreparedStatement {
     private String sql;
+    private MongoStatement statement;
 
     public MongoPreparedStatement(
             String sql, MongoConnection conn, String databaseName, boolean relaxed)
             throws SQLException {
-        super(conn, databaseName, relaxed);
         this.sql = sql;
+        this.statement = new MySQLStatement(conn, databaseName, relaxed);
```

Comment:
The design doc describes a slightly different approach where the constructor takes a MongoStatement object.  That would help in case we don't know here whether to construct a MySQL or MongoSQL statement

Reply:
I figured I'd leave that to [SQL-511: Finish MongoSQL Statement](https://jira.mongodb.org/browse/SQL-511) . @pmeredit did you have an idea/preference of which ticket would cover this change?

Reply:
Thanks for pointing that out.  I'm fine with it being handled in SQL-511.  👍

Reply:
sgtm